railties/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Raise an error when "recyclable cache keys" are being used by a cache store
+    that does not explicitly support it.
```

Comment:
Let's describe what cache stores need to do (implement `supports_cache_versioning` class method) to advertise support.

Reply:
For this what are you thinking? Maybe a section in the guide?